config/brakeman.ignore

```diff
@@ -3,50 +3,49 @@
     {
       "warning_type": "Mass Assignment",
       "warning_code": 105,
-      "fingerprint": "5011c2eef8a2a8400a7700cb5127ec34d988b403470b71484c7ea37f000d0669",
+      "fingerprint": "4aef20c77272b01619b5f67292f088db289359805fe0e21ef7bc05b6191d87ce",
       "check_name": "PermitAttributes",
       "message": "Potentially dangerous key allowed for mass assignment",
-      "file": "app/controllers/memberships_controller.rb",
-      "line": 33,
+      "file": "app/controllers/users_controller.rb",
+      "line": 68,
       "link": "https://brakemanscanner.org/docs/warning_types/mass_assignment/",
-      "code": "params.require(:membership).permit(:role)",
+      "code": "params[:filter].permit(:name, :email, :organisation_id, :role, :has_access)",
       "render_path": null,
       "location": {
         "type": "method",
-        "class": "MembershipsController",
-        "method": "membership_params"
+        "class": "UsersController",
+        "method": "filter_params"
       },
       "user_input": ":role",
       "confidence": "Medium",
       "cwe_id": [
         915
       ],
-      "note": "Changing role for Membership is allowed."
+      "note": ""
```

Comment:
Even though it's reasonably clear from the context, it feels like it would be useful to add a note to make it clear why we need this exception, and it probably good practice for us to always add a note when adding stuff to this file anyway.